tests/library/test_main_methods.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from recipe_scrapers import get_supported_urls, scraper_exists_for
+from recipe_scrapers._utils import get_host_name
+
+
+class TestMainMethods(unittest.TestCase):
+    def test_GetSupportedUrls(self):
```

Comment:
can you just change this and the follow up method to be snake_case `test_get_supported_urls` / `test_scraper_exists_for`